charts/linkerd2/values.yaml

```diff
@@ -75,7 +75,8 @@ global:
   controllerNamespaceLabel: linkerd.io/control-plane-ns
   linkerdNamespaceLabel: linkerd.io/is-control-plane
 
-
+#Add custom host here
+enforcedHostRegexp: ""
```

Comment:
This could be
Make sure there is a new line below to separate
```
# Enforced Host Validation Regular Expression
enforcedHostRegexp: ""

```

Reply:
For the comment, I think we should follow the casing practice we've had in most of the rest of the file (lower-case).

Reply:
Done